canonical-websites/www.ubuntu.com
# [UA-I] updates for /support/plans-and-pricing page

## Done

[List of work items including drive-bys]

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1FS1slJOfWXLmPob1eXrQ32DU4s6WP4oyGoT3t5E4W8Y/edit#)

## Issue / Card

Fixes #4873
